### PR TITLE
feat(nerdstorage): add a nerdstorage package and resource

### LINKED ISSUE #154: feat: Create a resource for nerdstorage
Based on the nerdstorage nerdgraph mutation category and the `actor.account.nerdStorage` query path.

### PR DESCRIPTION
**Pull request by ctrombley**
Resolves #154.

This PR adds nerdstorage support to the client.

### CODE REVIEW

File: pkg/nerdstorage/example_user_scope_test.go [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+	if err != nil {
+		log.Fatal("error marshalling NerdStorage document to json:", err)
+	}
+
+	err = json.Unmarshal(marshalled, &myDoc)
+	if err != nil {
+		log.Fatal("error unmarshalling NerdStorage document to struct:", err)
+	}
+
+	fmt.Printf("Document: %v\n", myDoc)
```
Comment (zlesnr) [inline pkg/nerdstorage/example_user_scope_test.go]: Dangling debug perhaps?  I see a few other printfs here.

Comment (zlesnr) [reply pkg/nerdstorage/example_user_scope_test.go]: Oh its a test.

File: pkg/nerdstorage/example_account_scope_test.go [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+func Example_accountScope() {
+	// Initialize the client configuration.  A Personal API key is required to
+	// communicate with the backend API.
+	cfg := config.New()
+	cfg.PersonalAPIKey = os.Getenv("NEW_RELIC_API_KEY")
+
+	// Initialize the client.
+	client := New(cfg)
+
+	accountID := 12345678
```
Comment (jthurman42) [inline pkg/nerdstorage/example_account_scope_test.go]: `os.Getenv("NEW_RELIC_ACCOUNT_ID")` ?

Comment (ctrombley) [reply pkg/nerdstorage/example_account_scope_test.go]: 👍

**Pull request merged**